include/openssl/x509v3.h

```diff
@@ -632,14 +632,24 @@ OPENSSL_EXPORT X509_EXTENSION *X509V3_EXT_nconf_nid(const CONF *conf,
 
 // X509V3_EXT_conf_nid calls |X509V3_EXT_nconf_nid|. |conf| must be NULL.
 //
-// TODO(davidben): This is the only exposed instance of an LHASH in our public
-// headers. cryptography.io wraps this function so we cannot, yet, replace the
-// type with a dummy struct.
+// TODO(davidben): This and |X509V3_EXT_conf| are the only exposed instance of
```

Comment:
Is this valid if we assert that `conf` is NULL in `X509V3_EXT_nconf`?

Reply:
ahh this was a typo on my side, it should be `X509V3_EXT_conf`

Reply:
I think what this comment's trying to say is that `X509V3_EXT_conf_nid` is the only function that exposes the `LHASH_OF(*)` type in it's argument. They wanted to deprecate this structure, but ran into libraries that were still consuming it. 

I updated this comment to include `X509V3_EXT_conf` since there are now two functions exposing `LHASH_OF(*)`. Both `X509V3_EXT_conf_nid` and `X509V3_EXT_conf` do the `NULL` assertion for `conf`.